src/parsers/js/transformers/eslint2/index.js

```diff
@@ -0,0 +1,23 @@
+import {defineRule, runRule} from '../../utils/eslintUtils';
+import pkg from 'eslint2/node_modules/eslint/package.json';
+
+const ID = 'eslint-v2';
+const name = 'ESLint v2'
+
+export default {
+  id: ID,
+  displayName: name,
+  version: pkg.version,
+  homepage: pkg.homepage,
+
+  defaultParserID: 'acorn-to-esprima',
+
+  loadTransformer(callback) {
+    require(['eslint2'], callback);
+  },
+
+  transform({ eslint, rules, sourceCode }, transformCode, code) {
+    defineRule(rules, transformCode);
+    return runRule(code, eslint, sourceCode);
```

Comment:
Well the issue has to do with how babel-eslint monkey patches ESLint - however we aren't running that so there aren't any issues (parseNoPatch instead of parse). This just means certain things won't work correctly like a scope rule dealing with flow types etc.